### PR TITLE
buffs clown honk virus

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
clown pda virus is buffed. to compensate, it starts with no charges, so the clown MUST slip targets to send virus

## Why It's Good For The Game
:o)
## Changelog
:cl:
balance: greatly buffs clown pda virus.
balance: clown pda virus starts with 0 charge- the clown must slip people to gain charges to use it
/:cl:

### CODE REVIEW

**Comment by USB3pt0** [comment]
speedmerge

**Comment by Sinestia** [comment]
pog

**Comment by Kapu1178** [comment]
speed merge

**Comment by Archanial** [comment]
This is very funny but it's too OOC for MRP and shouldn't use actual admin names.

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -745,8 +745,16 @@ GLOBAL_LIST_EMPTY(PDAs)
...
 
 	if (!silent)
-		playsound(src, 'sound/machines/twobeep_high.ogg', 50, 1)
-		audible_message("[icon2html(src, hearers(src))] *[ttone]*", null, 3)
+		if(honkamt > 9)//For clown virus.
+			var/adminname = pick(strings("admin_nicknames.json", "names", "config"))
+			honkamt -= 9
+			playsound(src, 'sound/effects/adminhelp.ogg', 50, 1)
+			audible_message("<font color='red' size='4'><b>-- Administrator private message --</b></font>", null, 3)
+			audible_message("<span class='danger'>Admin PM from-</span><span class='ooc'>[adminname]</span><span class='danger'>: [pick("What do you think you're doing?","Hey, got a minute?", "Care to explain just how you aren't powergaming right now?","We need to talk.","Stop that or I'll ban you.")]</span>", null, 3)
```
Comment (Archanial) [inline code/game/objects/items/devices/PDA/PDA.dm:753]:
```suggestion
			audible_message("<span class='danger'>Admin PM from-</span><span class='ooc'>[adminname]</span><span class='danger'>: [pick("What do you think you're doing?","Hey, got a minute?", "Care to explain just how you aren't powergaming right now?","We need to talk.","Stop that or I'll ban you.","What the fuck are you doing?")]</span>", null, 3)
```

**Comment by Sinestia** [comment]
> This is very funny but it's too OOC for MRP and shouldn't use actual admin names.

There are bonk sounds in-game and references to OOC stuff so it isn't that bad

**Comment by aidan-bowman** [comment]
> This is very funny but it's too OOC for MRP and shouldn't use actual admin names.

literally roundstart the clown can spam the funy admeme sound via sound synthesizer

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (zeskorion) [reply code/game/objects/items/devices/PDA/PDA.dm:753]: redundant

**Comment by Sinestia** [comment]
https://file.house/1eC3.png

fran meme quote when

File: code/game/objects/items/devices/PDA/PDA.dm (hunk 1 above)
Comment (Sinestia) [reply code/game/objects/items/devices/PDA/PDA.dm:753]: https://file.house/1eC3.png

make it this

**Comment by ItsTisme** [comment]
This is legit terrifying

**Comment by Archanial** [comment]
> > This is very funny but it's too OOC for MRP and shouldn't use actual admin names.
> 
> There are bonk sounds in-game and references to OOC stuff so it isn't that bad

No, it shows up as an actual admins message. This is really funny but bad idea in the long run.

**Comment by zeskorion** [comment]
> > > This is very funny but it's too OOC for MRP and shouldn't use actual admin names.
> > 
> > 
> > There are bonk sounds in-game and references to OOC stuff so it isn't that bad
> 
> No, it shows up as an actual admins message. This is really funny but bad idea in the long run.

No. It generates a fake admin name (like an admin using stealthmin tools)

**Comment by PowerfulBacon** [comment]
Admin messages are important, OOC and shouldn't be replicated in game outside of adminhelps themselves.

**Comment by OceanFish1** [comment]
No fun allowed momen

**Comment by zeskorion** [comment]
> No fun allowed momen

If you want it in go whine to ike or smth i spent like 10 minutrs on this

**Pull request closed without merging**